### ISSUE TITLE
[BUG] PlotNFT does not show

### ISSUE THREAD

**Issue by chinabala** [Chia-Network/chia-blockchain#8624]
When remove folders of .chia, and reinstall the main program...

Pool does not show my previously NFT pools. It's blank and ask me to create/join a pool.
(leave it for 24 hours and still the same)

So I then join a pool, it works and show all my NFT pools correctly. However... it adds up a new created pool under my account and I couldn't remove it,.

Is this a bug when reinstall chia program and pool doesn't show?
Should it add a feature to reload plot nft page?
Any chance to remove created NFT?

**Comment by shaneo257** [comment]
if you use the same mnemonics when you reinstalled, once your wallet has synced to the height when you created the nft it will then show

**Comment by chinabala** [comment]
> reinstalled

Yes. Using the same mnemonics and wallet has synced (ran more than 48 hours).
This can be reproduced, just deleted folders described above and reinstall. There's no way it can retrive NFT pool(s) back, other than create a new one...

**Comment by chinabala** [comment]
> if you use the same mnemonics when you reinstalled, once your wallet has synced to the height when you created the nft it will then show

Yes. Using the same mnemonics and wallet has synced (ran more than 48 hours).
This can be reproduced, just deleted folders described above and reinstall. There's no way it can retrive NFT pool(s) back, other than create a new one...

**Comment by loppefaaret** [comment]
please be careful with the amount of plotNFT's you create! at some point it could have a negative impact on the current perfomance of the wallet

**Comment by spamkutu** [comment]
Yes its weird bug and that happen almost always you have to add new plotNTFs to view and reach yours NTFS but that time and each time you had add 1 plus NFT  that must be real solve Without any add plotntfs dosent show with that add you must have 1 plus nfts and you cannot control and also if that one not showed or deleted your all plots poooof.

Without  add a new plotNFT each time on reinstall or just formated system it must be seen by itself  with that way i made accidently 30-35 some poolls of course all diff ntfs and like loppefaaret told its do negative impact on current performance and its only show 18 ntfs max on here futhers you also paid for all this time its use only 1 mojo but on next who ever know how much money you wasted with that way,

this must be solve realy bug.

**Comment by defaultsecurity** [comment]
I can't believe this has not been fixed yet. It wastes resources on the client side and on the blockchain.
